go/host/node/node.go

```diff
@@ -449,19 +449,20 @@ func (a *Node) processBlocks(blocks []common.EncodedBlock, interrupt *int32) err
 	}
 
 	// Nodes can start before the genesis was published, and it makes no sense to enter the protocol.
-	if result.ProducedRollup.Header != nil {
```

Comment:
The PR changes nothing, except flipping this condition around to avoid indenting the whole block and make the codeflow clearer.